platform/linux-generic/m4/odp_pcap.m4

```diff
@@ -9,7 +9,7 @@ AC_CHECK_HEADER(pcap/pcap.h,
 [])
 
 if test "$have_pcap" = "yes"; then
-    ODP_CFLAGS="$AM_CFLAGS -DHAVE_PCAP"
+    AC_DEFINE([HAVE_PCAP], 1, [Define to 1 if you have pcap library])
```

Comment:
"Define to 1 if you have pcap library" has to be "Defined to 1 if pcap library found"